atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/LockWatchValueScopingCacheImpl.java

```diff
@@ -267,8 +267,8 @@ private synchronized void clearCache(
                 "Clearing all value cache state",
                 SafeArg.of("currentVersion", currentVersion),
                 SafeArg.of("latestUpdateFromUpdate", latestVersionFromUpdate),
-                SafeArg.of("firstEventSequence", firstEvent),
-                SafeArg.of("lastEventSequence", lastEvent));
+                SafeArg.of("firstEventSequence", Optional.ofNullable(firstEvent).map(LockWatchEvent::sequence)),
+                SafeArg.of("lastEventSequence", Optional.ofNullable(lastEvent).map(LockWatchEvent::sequence)));
```

Comment:
Should `LockWatchEvent` be annotated as `@Unsafe`?

Reply:
Seems like `LockToken` should be `@Unsafe` given the token can be arbitrary `byte[]`

https://github.com/palantir/atlasdb/blob/ba7f38e4bf9260f330f6c0e8643c2133963eada3/lock-api-objects/src/main/java/com/palantir/lock/LockDescriptor.java#L35-L48